deploy/operator/base/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         runAsNonRoot: true
       containers:
         - name: hacbs-jvm-operator
-          image: hacbs-jvm-operator
+          image: hacbs-jvm-operator:latest
```

Comment:
I'm not sure that would help as https://docs.kubelinter.io/#/generated/checks?id=latest-tag would still complain? Would it not be better to exclude this check in the configuration file?

Reply:
Changed tag name as @stuartwdouglas suggested.